packs/feats/archers-aim.json

```diff
@@ -23,7 +23,40 @@
                 }
             ]
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "archers-aim",
+                    "ranged"
+                ],
+                "selector": "attack",
+                "type": "circumstance",
+                "value": 2
+            },
+            {
+                "key": "Note",
+                "predicate": [
+                    "archers-aim",
+                    "ranged",
+                    "target:condition:concealed"
+                ],
+                "selector": "attack",
+                "text": "Ignore your prey's concealed condition.",
+                "title": "{item|name}"
+            },
+            {
+                "key": "Note",
+                "predicate": [
+                    "archers-aim",
+                    "ranged",
+                    "target:condition:hidden"
+                ],
+                "selector": "attack",
+                "text": "Reduce the flat check from being hidden to 5.",
+                "title": "{item|name}"
+            }
```

Comment:
```suggestion
            {
                "key": "FlatModifier",
                "predicate": [
                    "archers-aim",
                    "ranged"
                ],
                "selector": "bow-weapon-group-attack",
                "type": "circumstance",
                "value": 2
            },
            {
                "key": "Note",
                "predicate": [
                    "archers-aim",
                    "ranged",
                    "target:condition:concealed"
                ],
                "selector": "bow-weapon-group-attack",
                "text": "PF2E.SpecificRule.AttackEffects.IgnoreConcealed",
                "title": "{item|name}"
            },
            {
                "key": "Note",
                "predicate": [
                    "archers-aim",
                    "ranged",
                    "target:condition:hidden"
                ],
                "selector": "bow-weapon-group-attack",
                "text": "Reduce the flat check from being hidden to 5.",
                "title": "{item|name}"
            }
```
We need to localize the flat check reduction note so translators can reach those strings. Add the new strings to re-en.json under the language files in the system, I think putting it under AttackEffects as `ReduceHiddenCheck` or something should be fine. Ping me in discord (Tikael) if you need help with that.